Adds Mexican tourists to the restaurant system, plus some QOL tweaks.

ArcaneMusic opened the pull request:
## About The Pull Request

Alright, brass tacks, adds new Mexican tourists to the restaurant. Their orders are fairly traditional spanish and mexican food, their dialog sound is some pretty classy spanish guitar, and by god they care about football. REAL football.

This also created a few QOL tweaks that I noticed and changed along the way:

- Chef Holosign creator for restaurant seating didn't look very unique due to using the standard holosign sprite, so I gave it a nice recolor with a bright white and red restaurant pattern on an edge to help it stand out a bit.
- The two taco recipies have been given a minor name tweak, which actually already existed in the crafting menu, but didn't change the names of the two tacos, so now the actual food tacos are classic tacos ( Cheese, meat, cabbage, tortilla) and plain tacos (tortilla, cheese, meat).

![Mexibot](https://user-images.githubusercontent.com/41715314/110050060-1a166480-7d21-11eb-8fed-3e261a1bac94.gif)
_The robot sprite._

## Why It's Good For The Game

Adds more variety to the restaurant system, keeps it tasteful in flavor and the foods themselves aren't cooked very often on station, so this helps bump up food variety on the whole.
Helps to distinguish the existing food on station a bit with the variable name, and helps to make the venue specific aspects of the restaurant system stand out a tiny bit more.

## Changelog
:cl:
add: There are now space tourists from south of the border. Andele!
qol: The two taco recipes are now distinguished by name in game to help tell the two apart.
imageadd: The Restaurant Holoseat Creator now uses a slightly different sprite to help it stand out.
/:cl:


Demo Video (Note: this was the exact moment I realized I needed to change the name):
https://discord.com/channels/326822144233439242/326831214667235328/817188033446412290

ArcaneMusic committed:
Adds mexican tourists to the restaurant system.

tralezab commented:
damn NICE

Ghommie commented on code/modules/food_and_drinks/restaurant/customers/_customer.dm:
```diff
@@ -125,3 +125,20 @@
+	/datum/venue/bar = list(/datum/reagent/consumable/ethanol/whiskey = 6, /datum/reagent/consumable/ethanol/tequila = 20, /datum/reagent/consumable/ethanol/tequila_sunrise = 1, /datum/reagent/consumable/ethanol/beer = 15, /datum/reagent/consumable/ethanol/patron = 5, /datum/reagent/consumable/ethanol/brave_bull = 5, /datum/reagent/consumable/ethanol/margarita = 8))
+
+
+	found_seat_lines = list("Como te va, space station 13?", "Who's ready to party!", "Ah, muchos gracias.", "Ahhh, smells like mi abuela's cooking!")
```
```suggestion
	found_seat_lines = list("¿Como te va, space station 13?", "Who's ready to party!", "Ah, muchas gracias.", "Ahhh, smells like mi abuela's cooking!")
```

Ghommie commented on code/modules/food_and_drinks/restaurant/customers/_customer.dm:
```diff
@@ -125,3 +125,20 @@
+
+
+	found_seat_lines = list("Como te va, space station 13?", "Who's ready to party!", "Ah, muchos gracias.", "Ahhh, smells like mi abuela's cooking!")
+	cant_find_seat_lines = list("En Serio? Seriously, no seats?", "Andele! I want a table to watch the football match!", "Ay Caramba...")
```
```suggestion
	cant_find_seat_lines = list("¿En Serio? Seriously, no seats?", "Andele! I want a table to watch the football match!", "Ay Caramba...")
```

Ghommie commented on strings/names/mexican_prefix.txt:
```diff
@@ -0,0 +1,13 @@
+Gonzales
+Rio
+Pele
+Conqustador
```
That's a typo.
```suggestion
Conquistador
```

Ghommie commented on code/modules/food_and_drinks/restaurant/customers/_customer.dm:
```diff
@@ -125,3 +125,20 @@
+
+	found_seat_lines = list("Como te va, space station 13?", "Who's ready to party!", "Ah, muchos gracias.", "Ahhh, smells like mi abuela's cooking!")
+	cant_find_seat_lines = list("En Serio? Seriously, no seats?", "Andele! I want a table to watch the football match!", "Ay Caramba...")
+	leave_mad_lines = list("Aye dios mio, I'm out of here.", "Esto es ridiculo! I'm leaving!", "I've seen better cooking at taco campana!", "I though this was a restaurant, pero es porqueria!")
```
```suggestion
	leave_mad_lines = list("Aye dios mio, I'm out of here.", "Esto es ridículo! I'm leaving!", "I've seen better cooking at taco campana!", "I though this was a restaurant, pero es porquería!")
```

Ghommie commented on strings/names/mexican_prefix.txt:
```diff
@@ -0,0 +1,13 @@
+Ranchero
+Senor
```
```suggestion
Señor
```

Ghommie reviewed (changes requested):
Bettering this hollywood spanish a little.

Pugie commented:
Sure, boss, this sounds good.

ArcaneMusic committed:
Apply suggestions from code review

I forgot if unicode breaks things but I have since been informed it doesn't. Thanks Ghommie.

Co-authored-by: Ghom <42542238+Ghommie@users.noreply.github.com>

maxymax13 commented:
all these restaurant costumer prs never give any love to the bartender part

ArcaneMusic commented:
> all these restaurant costumer prs never give that much love to the bartender part

Any suggestions to keep it more neutral? Admittedly I probably lean closer towards food than drinks here.

tralezab commented:
I think that's something qust is gonna have to address at some time in the future in general, every single customer is like that
